lib/bddsm/suite.rb

```diff
@@ -11,7 +11,15 @@ def initialize
       result.subscribe report
     end
 
-    def report = @report ||= BDDSM::ProgressReport.new(result:, io: $stdout)
+    def run
+      load_files
+      run_describes
```

Comment:
I don't think, thank every test should be scoped into describe block. It could be scenario or something else. That's why I named it `run_examples` - more abstract